src/cephadm/cephadm.py

```diff
@@ -4497,6 +4502,12 @@ def _add_deploy_parser_args(
         default=[],
         help='Additional entrypoint arguments to apply to deamon'
     )
+    parser_deploy.add_argument(
+        '--termination-grace-period-seconds',
+        type=int,
+        default=None,
+        help='Graceful stop timeout written to unit.stop'
```

Comment:
I think it's better to explain what the parameter is about (not what we do with it). So I'd suggest something like `Time in seconds to wait for graceful service shutdown before forcefully killing it`